packages/tooling/adaptive-ui-explorer/src/components/adaptive-component.ts

```diff
@@ -0,0 +1,100 @@
+import {
+    css,
+    customElement,
+    ElementViewTemplate,
+    FASTElement,
+    html,
+    observable,
+} from "@microsoft/fast-element";
+import { CSSDesignToken } from "@microsoft/fast-foundation";
+import { Swatch } from "@microsoft/adaptive-ui";
+
+function template<T extends AdaptiveComponent>(): ElementViewTemplate<T> {
+    return html<T>`
+        <template tabindex="0">
+            <style>
+                :host {
+                    background: ${x => x.fillRestToken?.createCSS()};
```

Comment:
Whats the reason for putting this css in an HTML template and not a stylesheet for the component?

Reply:
Each instance of the `adaptive-component` can have a different set of tokens applied to it, so all `adaptive-component`s don't share the same stylesheet. This may be similar to recreating `Behavior` support. Is there a better way I can achieve this?
I had to call `createCSS` directly otherwise I was getting "[object Object]" in the output, I believe because it's not going through the css rendering which resolves the design tokens.

Reply:
Oh gotcha, yeah you would need CSS bindings for that. `ElementStyles` looks for CSS directives and calls createCSS itself, which is why you need to call it here and not there.

Reply:
Quick thought. Create a CSS variable to use in your stylesheet. Then add a style binding on the html element and use that to set the variable's value.